earth_enterprise/src/fusion/autoingest/AssetSerializer_unittest.cpp

```diff
@@ -27,28 +27,43 @@ using namespace khxml;
 // a file, and we don't want to do that in unit tests. We'll cover the happy
 // path plenty of times in integration and other tests.
 
+bool AssetThrowPolicy::allow_throw(true);  // Force Load to throw exceptions
+
 class TestItemStorage {};
 
 class TestItem : public TestItemStorage {
   public:
     using Base = TestItemStorage;
     static int nextValue;
     static string loadFile;
+    static string xmlFilename;
+    static bool newInvalidCalled;
     int val;
+    uint64 timestamp; 
+    time_t filesize;
```

Comment:
I think these types need to be switched for **timestamp** and **filesize**

Reply:
D'oh! Good catch.